plugins/Live/Controller.php

```diff
@@ -79,6 +79,39 @@ public function getLastVisitsStart()
         $view->idSite = $this->idSite;
         $api = new Request("method=Live.getLastVisitsDetails&idSite={$this->idSite}&filter_limit=10&format=php&serialize=0&disable_generic_filters=1");
         $visitors = $api->process();
+        $visits = array();
+        foreach ($visitors as $visitor) {
+            $actions = $visitor['actionDetails'];
+
+            foreach ($actions as $action) {
+                if ($action['type'] != "ecommerceOrder" && $action['type'] != "ecommerceAbandonedCart") {
```

Comment:
I wonder why they are excluded? Do they not contain a URL? I'd maybe also check for `!empty($action['url'])` as there might be some actions that do not contain a URL. eg when tracking mobile apps or when Piwik is enriched by 3rd party plugins maybe.